src/pages/Swap/AdvancedSwapMode/Trade/Trade.utils.ts

```diff
@@ -0,0 +1,11 @@
+import { formatDistance, subDays } from 'date-fns'
+
+export const formatDate = (timestamp: number) => {
```

Comment:
`formatDistance` returns the distance between the given dates in words.

maybe `formatDistanceDate` or something like that is more descriptive of what it does than `formatDate`

```suggestion
export const formatDistanceDate = (timestamp: number) => {
```